masonite/packages.py

```diff
@@ -47,6 +47,17 @@ def append_web_routes(location):
     print('\033[92mroutes/web.py File Appended!\033[0m')
 
 
+def append_file(from_location, to_location):
+    from_fp = open(from_location, "r")
+    to_fp = open(os.path.join(os.getcwd(), to_location), 'a')
+
+    to_fp.write(from_fp.read())
```

Comment:
Why can't we just move file here?

Reply:
sometimes we will need to append it. Think of the routes.py file. Can't just move the file, the contents of it needs to be appended at the end

Reply:
Got it, thanks.

Reply:
Which means "fp" at the end of variables' name?

Reply:
file pointer.

I hate making short variable names lol. I should make them longer and spell out file_pointer